packit_service/worker/handlers/distgit.py

```diff
@@ -479,9 +495,14 @@ def run(self) -> TaskResults:
             self.job_config,
             downstream_local_project=self.local_project,
         )
+        branch = (
+            self.project.get_pr(self.data.pr_id).source_branch
```

Comment:
we are interested in the branch the changes were merged into, so I think this should be the `target_branch`

Reply:
Ok, thanks. I was not sure. Just a thing I do not understand: if I do checkout the `target_branch` I think I do not have all the commits in the PR and thus I perform a koji-build with not all commits. Am I wrong or this is ok for the koji-build we want?